infrastructure/terraform/components/reporting/module_powerbi_gateway_vpc.tf

```diff
@@ -0,0 +1,83 @@
+module "powerbi_gateway_vpc" {
+
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "5.5.1"  # Adjust to the latest version
+
+  create_vpc = var.enable_powerbi_gateway
+
+  name = "${local.csi}-powerbi-gateway-vpc"
+  cidr = "10.0.0.0/16"
+
+  azs             = var.availability_zones
+  public_subnets  = var.public_subnet_cidrs
+  private_subnets = var.private_subnet_cidrs
+
+  enable_nat_gateway = true
+  single_nat_gateway = false
+  enable_dns_support = true
+  enable_dns_hostnames = true
+  create_igw = true
+
+  tags = {
```

Comment:
don't we apply tags at provider level, probs don't need it here?

Reply:
fair point - i think i'd thought they were required to add a name to these resources at all